iec61850serversimulator/src/test/java/com/cgi/iec61850serversimulator/SwitchingMomentCalculatorTest.java

```diff
@@ -17,17 +18,103 @@
 	 * JUnit test class which tests the various related functions of the
 	 * SwitchingMomentCalculator class.
 	 *
-	 * This includes unique functions for mocking a
-	 * relay and schedule.
+	 * This includes unique functions for mocking a relay and schedule.
 	 *
-	 * @formatter:off
 	 * Tests include:
-	 * - Checking the mocked relay
-	 * - Checking the mocked schedule
-	 * - Checking the generated Switching Moment made out of a mocked relay and schedule
-	 * @formatter:on
+	 * <ul>
+	 * <li>Checking the mocked relay
+	 * <li>Checking the mocked schedule
+	 * <li>Checking the generated Switching Moment made out of a mocked relay and
+	 * schedule
+	 * </ul>
 	 */
 
+	public static class MockedScheduleBuilder {
```

Comment:
This creates a normal Schedule object. Move the "MockedScheduleBuilder" to the Schedule class (and rename it to ScheduleBuilder).